cabal.project

```diff
@@ -54,6 +54,9 @@ package bitvec
 package plutus-scripts-bench
   haddock-options: "--optghc=-fplugin-opt PlutusTx.Plugin:defer-errors"
 
+constraints:
+  io-classes-mtl < 0.1.2.0
```

Comment:
Why? can you add a comment explaining why?

Reply:
If you don't include it you get this error:

```
src/Control/Monad/Class/MonadSTM/Trans.hs:116:5: error: [GHC-54721]
    ‘writeTMVar’ is not a (visible) method of class ‘MonadSTM’
    |
116 |     writeTMVar     = ContTSTM .: writeTMVar
    |     ^^^^^^^^^^

src/Control/Monad/Class/MonadSTM/Trans.hs:194:5: error: [GHC-54721]
    ‘writeTMVar’ is not a (visible) method of class ‘MonadSTM’
    |
194 |     writeTMVar     = lift .: writeTMVar
    |     ^^^^^^^^^^

src/Control/Monad/Class/MonadSTM/Trans.hs:272:5: error: [GHC-54721]
    ‘writeTMVar’ is not a (visible) method of class ‘MonadSTM’
    |
272 |     writeTMVar     = lift .: writeTMVar
    |     ^^^^^^^^^^

src/Control/Monad/Class/MonadSTM/Trans.hs:350:5: error: [GHC-54721]
    ‘writeTMVar’ is not a (visible) method of class ‘MonadSTM’
    |
350 |     writeTMVar     = lift .: writeTMVar
    |     ^^^^^^^^^^

src/Control/Monad/Class/MonadSTM/Trans.hs:428:5: error: [GHC-54721]
    ‘writeTMVar’ is not a (visible) method of class ‘MonadSTM’
    |
428 |     writeTMVar     = lift .: writeTMVar
    |     ^^^^^^^^^^

src/Control/Monad/Class/MonadSTM/Trans.hs:506:5: error: [GHC-54721]
    ‘writeTMVar’ is not a (visible) method of class ‘MonadSTM’
    |
506 |     writeTMVar     = lift .: writeTMVar
    |     ^^^^^^^^^^

src/Control/Monad/Class/MonadSTM/Trans.hs:584:5: error: [GHC-54721]
    ‘writeTMVar’ is not a (visible) method of class ‘MonadSTM’
    |
584 |     writeTMVar     = lift .: writeTMVar
    |     ^^^^^^^^^^

src/Control/Monad/Class/MonadSTM/Trans.hs:662:5: error: [GHC-54721]
    ‘writeTMVar’ is not a (visible) method of class ‘MonadSTM’
    |
662 |     writeTMVar     = lift .: writeTMVar
    |     ^^^^^^^^^^
```

This is just changes in the definition of type classes in our dependencies that break other dependencies.  When all our other dependencies update accordingly we can remove this constraint.